v2/setup/images-proxy.sh

```diff
@@ -14,7 +14,7 @@ if [ "$(etcdctl get images-proxy-bootstrapped)" == "true" ]; then
 fi
 etcdctl set images-proxy-bootstrapped true
 
-etcdctl set /images/capcom       "behance/capcom:latest"
+etcdctl set /images/capcom       "behance/capcom:8fa735d310d0a46fc3acc34245c8bf63e8447ef4"
 etcdctl set /images/capcom2      "behance/capcom:latest"
```

Comment:
I was going to comment on this as well @ko-be ... I know that `images/capcom2` is set to be `behance/capcom:latest` on the currently running cluster, but we should confirm that's *what it should be set to*, or if we should lock the two containers to the same sha, ask maybe Iz....